contracts/reviewer.sol

```diff
@@ -29,22 +52,60 @@ contract ReviewerTransactionGuard is BaseGuard {
         _;
     }
 
-    address public ApplicationReviewRegistryAddress = address(); // Enter Registry Address according to the chain this contract is deployed on
-    address public ApplicationRegistryAddress = address(); // Enter Registry Address according to the chain this contract is deployed on
-    address public WorkspaceRegistryAddress = address(); // Enter Registry Address according to the chain this contract is deployed on
+    modifier onlyOwner() {
+        require(msg.sender == owner, "Unauthorised: Not the owner");
+        _;
+    }
+
+    address public ApplicationReviewRegistryAddress;
+
+    IApplicationRegistry public applicationReg;
+    IApplicationReviewRegistry public applicationReviewReg;
+    IWorkspaceRegistry public workspaceReg;
+
+    address public owner;
     address public safeAddress;
     address[] public reviewers;
     uint96 public threshold;
     bytes4 public constant removeGuardBytesData = hex"e19a9dd9";
     bytes4 public constant multiSendBytesData = hex"8d80ff0a";
+
     // bytes4 public constant ENCODED_SIG_SET_GUARD = bytes4(keccak256("setGuard(address)")); // hex"e19a9dd9"
     // bytes4 public constant ENCODED_SIG_MULTI_SEND = bytes4(keccak256("")); // hex"8d80ff0a"
 
-    constructor(address _safeAddress, address[] memory _reviewers, uint96 _threshold) {
-        require(_reviewers.length >= _threshold, "Threshold can't be greater than the number of reviewers");
+    constructor(
```

Comment:
The addresses of the 3 interfaces can be passed in the constructor itself as it would avoid calling three functions after the deployment of the contract.